backend/services/wallet/src/routes.py

```diff
@@ -0,0 +1,5 @@
+from src import app
+
+@app.route("/")
+def index():
+    return "Hello from your wallet"
```

Comment:
We need endpoints for adding a new wallet and getting a wallet's amount, as well as adding/subtracting from a wallet. All will access the wallet through a walletId passed in the request. Please refer to the Blockchain endpoints, which these endpoints will simply be hitting.

Reply:
done